Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.5
+FROM bitwalker/alpine-erlang
```

Comment:
@dropfen How else do you plan to run erlang binary code without the erlang VM...